syntax = "proto3";
package newqueue;

service NewQueue {
    rpc GetNext (GetNextRequest) returns (GetNextResponse) {}
}

message GetNextRequest {
    int32 limit = 1;
    int32 training_id = 2;
}

message GetNextResponse {
    repeated string keys = 2;
}
